Sources/SKCore/LanguageServer.swift

```diff
@@ -80,20 +80,29 @@ open class LanguageServerEndpoint {
     // Do nothing.
   }
 
-  open func _logRequest<R>(_ request: Request<R>) {
-    logAsync { _ in
-      "\(type(of: self)): \(request)"
+  open func _logRequest<R>(_ request: Request<R>, method: String) {
```

Comment:
`R.method` (or `N.method` in the notification case) should give you the method string.  Only response should need this, since it doesn't statically know what the request type.

Reply:
Done, thanks.